tests/timesignature_tests.ts

```diff
@@ -17,6 +17,8 @@ const TimeSignatureTests = {
     const run = VexFlowTests.runTests;
     run('Basic Time Signatures', basic);
     run('Big Signature Test', big);
+    run('Additive Signature Test', additive);
```

Comment:
Can we add a "Bad Signatures Test" to verify the corner cases of the parsing?

Reply:
Could you suggest some cases that you would like to see?

Reply:
`+`, `1+`, `+1`, `(3+` , `+3)`, `()`, `(+)`, `(4/5)`, `(4+3/)5`, etc.

Lots of edge cases here.

Reply:
I have included this cases in the parser test.
- `x` is valid
- `(4/5)` `(4+3/)5` are also valid using the simple regex. I think that this is no problem as the user will identify that the rendered result is not right. I left them out.